rundeckapp/grails-app/services/rundeck/services/TourLoaderService.groovy

```diff
@@ -30,6 +27,22 @@ class TourLoaderService {
                 tourManifest.add([provider:prov.key,loader:manifest.name ?: title ?: prov.key,tours:ungroupedTours])
             }
         }
+
+        pluginService.listPlugins(TourLoaderPlugin).each { prov ->
+            TourLoaderPlugin tourLoader = pluginService.configurePlugin(prov.key, tourLoaderProviderService, frameworkService.getFrameworkPropertyResolver(project), PropertyScope.Instance).instance
+            def title = getPluginTitle(tourLoader)
+            def manifest = tourLoader.tourManifest
```

Comment:
Perhaps: 
```
def manifest = project ? tourLoader.getTourManifest(project) : tourLoader.tourManifest
loadTours(manifest, title, prov)
```

Reply:
same as the previous comment, I will double-check if I can deal with that on the implemented plugin